ext/opcache/jit/zend_jit_trace.c

```diff
@@ -7886,7 +7898,7 @@ exit:;
 	return ret;
 }
 
-int ZEND_FASTCALL zend_jit_trace_hot_side(zend_execute_data *execute_data, uint32_t parent_num, uint32_t exit_num)
+static int zend_jit_trace_hot_side(zend_execute_data *execute_data, uint32_t parent_num, uint32_t exit_num)
```

Comment:
This function is only called from pure C code, so let the compiler decide on the calling convention. Since there's only one caller, a good compiler will inline it completely, making the choice of calling convention disappear completely.
Initially, I removed ZEND_FASTCALL because the calling convention doesn't matter for static functions, while it may matter for exported functions; but I was wrong and accidently changed only function called from assembler code; I got confused because all assembler code gets `#include`d in in PHP's JIT, i.e. living in the same compilation unit.